initial personas draft (#1824)
operator-framework/operator-controller: docs/draft/project/personas.md

> OLM Personas

Reviewer: ```suggestion
# OLM Personas
```
Can we start the title with #

> # About this doc

Reviewer: ```suggestion
```

The overview info, what is the doc about is added in all docs after the title. So, I do not think that we need the sub-section title

This document attempts to identify essential roles in the OLM lifecycle and associate the duties logically performed by each role. Though some roles can be (and even may typically be) performed by the same actor, they are logically distinct roles with different goals.

> # Framework
> OLM roles are broadly categorized here as **Producers** or **Consumers**, depicting whether that role typically is producing content for use in the ecosystem or is using (consuming) content. 

Reviewer: ```suggestion
OLM roles are broadly categorized here as **Producers** or **Consumers**, depicting whether that role typically produces content for use in the ecosystem or uses (consuming) content. 

Therefore, the personas are grouped by:
- **Consumers** – Users who consume or interact with the content managed by OLM.
- **Producers** – Users who produce content for OLM, including cluster extensions or catalogs.
```
I don't think we need the sub-title here, as it already feels like part of the overview. If we do want to add something, perhaps 'Group of Personas' would be a better fit. Also, 'Framework' might not be the most intuitive term for categorizing groups.
